lib/mix/test/mix/tasks/release_test.exs

```diff
@@ -297,7 +298,7 @@ defmodule Mix.Tasks.ReleaseTest do
         assert String.trim_trailing(version) == "eval 0.1.0"
         refute File.exists?(Path.join(root, "RELEASE_BOOTED"))
 
-        {hello_world, 0} = System.cmd(script, ["eval", ~s[IO.puts :hello_world]])
+        {hello_world, 0} = System.cmd(script, ["eval", "IO.puts(:hello_world)"])
```

Comment:
Just to be clear, is this a style change or an actual fix? :)

Reply:
Using parentheses (and removing the space) is an actual fix 🙂 

Replacing `~s[...]` with `"..."` is a style change 😬

Reply:
If removing the spaces is an actual thing, then we have a bug in our batch script/evaluator, because it means it is not handling spaces accordingly. `IO.puts :hello_world` should be a single command, because we are passing it as a single command. So perhaps we should leave it with space to help us find bugs?

Reply:
Which error would you get? `IO.puts/0` does not exist or similar?

Reply:
I got to the same conclusion and I was currently investigating the bug :)

Reply:
Here is the error:

![Screen Shot 2019-05-21 at 20 20 57](https://user-images.githubusercontent.com/651203/58120684-03635400-7c06-11e9-8ed2-c57c11582597.png)

Reply:
I think it's not related to releases:

![Screen Shot 2019-05-21 at 21 10 46](https://user-images.githubusercontent.com/651203/58123911-35c47f80-7c0d-11e9-98d8-231871d85efe.png)

Reply:
@fertapric does it change if you give it an absolute path from the beginning?

Reply:
@fertapric and are you using powershell or cmd?

Reply:
Also, which OTP version?

Reply:
> @fertapric does it change if you give it an absolute path from the beginning?

Same error apparently.

![Screen Shot 2019-05-21 at 22 03 29](https://user-images.githubusercontent.com/651203/58126862-49bfaf80-7c14-11e9-8882-1c5021635e6d.png)

> @fertapric and are you using powershell or cmd?

Powershell.

> Also, which OTP version?

OTP 21.0.1